src/js/node/net.ts

```diff
@@ -329,7 +331,8 @@ const Socket = (function (InternalSocket) {
         } else {
           self.authorized = true;
         }
-        self.server.emit("secureConnection", self);
+        server[bunSocketServerOptions]?.connectionListener?.$call(server, self);
```

Comment:
If connectionListener is undefined, this will error

is that intentional?

Reply:
was not intentional fixed in https://github.com/oven-sh/bun/pull/14695/commits/bf3f328a659189400894deca043032a91a66f71f